Source/sources/Castro_thermo.cpp

```diff
@@ -7,7 +7,8 @@ void
 Castro::construct_old_thermo_source(MultiFab& source, MultiFab& state, Real time, Real dt)
 {
   // we only include p divU in method of lines integration
-  if (time_integration_method == CornerTransportUpwind) return;
+  if (time_integration_method == CornerTransportUpwind ||
+      time_integration_method == SimplifiedSpectralDeferredCorrections) return;
```

Comment:
Same comment here about returning the logic: I would prefer
```
  if (!(time_integration_method == MethodOfLines ||
       time_integration_method == SpectralDeferredCorrections)) return;
```

Reply:
done